lib/Backends/CPU/CPUDeviceManager.cpp

```diff
@@ -70,9 +73,9 @@ void CPUDeviceManager::evictNetworkImpl(const Module *module) {
     return;
   }
 
-  FunctionMapTy moduleFuncs = std::move(modIt->second);
-  for (const auto &func : moduleFuncs) {
-    functions_.erase(func.first);
+  std::set<std::string> moduleFuncs = std::move(modIt->second);
```

Comment:
nit: not really need to move construct, just take a ref `const auto & moduleFuncs = modIt->second;`